### ISSUE TITLE
Very high number of translocations called in FFPE TNBC genomes

### ISSUE THREAD

**Issue by orsolyasipos** [PapenfussLab/gridss#496]
Hi, 

I have run GRIDSS on 24 FFPE TNBC tumour samples and used the somatic filtering script to filter out relevant SVs. 

When I created rearragement catalogues based on the filtered SVs (rearragement signature extraction as a goal in mind), I noticed that the most prevalent SVs were non-clustered translocations in all but 1 samples. 

I was wondering if you have ever encountered when using GRIDSS this kind of bias in FFPE samples when calling somatic SVs.

Many thanks for the advice, 
Orsolya

**Comment by d-cameron** [comment]
Yes. FFPE samples can have a very high FDR. FFPE artifacts will not be PON filtered as the Hartwig GRIDSS PON is based on sequencing of fresh frozen samples thus does not include any systematic FFPE false positives.

We have done some FFPE vs FF sequencing and are currently in the processing of seeing if we can adjust the filtering criteria to include removal of FFPE artifect. Unfortunately, the analysis part of this project has only just started so I don't have any recommendations that I know will work.

If you have matched normals for your samples I recommended adding these to the PON so you can at least remove the systematic FFPE artifacts. See https://github.com/PapenfussLab/gridss#how-do-i-merge-pons.

**Comment by orsolyasipos** [comment]
Thanks very much for the advice on adding the matched normals to see how many false positives can be removed.
